src/app/containers/Ad/Amp/index.jsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
 import {
   AMP_ACCESS_JS,
   AMP_ADS_JS,
 } from '@bbc/psammead-assets/amp-boilerplate';
 import {
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
 } from '@bbc/gel-foundations/breakpoints';
-import {
-  GEL_SPACING_TRPL,
-  GEL_SPACING_QUAD,
-  GEL_SPACING_DBL,
-  GEL_SPACING,
-} from '@bbc/gel-foundations/spacings';
-import { C_LUNAR_LIGHT } from '@bbc/psammead-styles/colours';
 
-const StyledWrapper = styled.div`
+import { GEL_SPACING_DBL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { C_LUNAR_LIGHT, C_RHINO } from '@bbc/psammead-styles/colours';
+import pathOr from 'ramda/src/pathOr';
+import { getMinion } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const FullWidthWrapper = styled.div`
   background-color: ${C_LUNAR_LIGHT};
 `;
 
-const StyledAd = styled.div`
-  /* To centre page layout for Group 4+ */
-  margin: 0 auto;
-  width: 100%; /* Needed for IE11 */
+const StyledWrapper = styled.div`
+  margin: 0 auto; /* To centre page layout for Group 4+ */
   text-align: center;
-  padding-top: ${GEL_SPACING_TRPL};
   padding-bottom: ${GEL_SPACING};
 
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    padding-top: ${GEL_SPACING_QUAD};
     padding-bottom: ${GEL_SPACING_DBL};
     max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
   }
 `;
 
+const StyledAd = styled.div`
+  display: inline-block;
+`;
+
+const StyledLink = styled.a`
+  ${({ script }) => script && getMinion(script)};
+  ${({ service }) => getSansRegular(service)}
+  color: ${C_RHINO};
+  margin: 0; /* Reset */
+  text-decoration: none;
+  text-transform: uppercase;
+  display: block;
+  padding: ${GEL_SPACING} 0;
+  
+  text-align: ${({ dir }) => (dir === 'ltr' ? `right` : `left`)};
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding-top: ${GEL_SPACING_DBL};
+  }
+  
+  &:hover,
+  &:focus {
```

Comment:
We don't need the `focus` selector if we are hiding the link from the tab order.